__tests__/lotto.test.js

```diff
@@ -0,0 +1,49 @@
+import { generateLottoNumber, getTicket } from '../src/lotto.js'
+import { LottoError } from '../src/errors/lottoError.js'
+
+describe('getTicket 함수 테스트', () => {
+  it('1,000원을 입력하면 1장을 발급한다', () => {
+    const money = 1000
```

Comment:
`lottoGame.test.js`와 일관성을 맞추시면 좋겠어요!
천 단위 구분 기호를 사용해 맞춰 주세요 😊

```suggestion
    const money = 1_000
```